docs/pages/access-controls/guides/moderated-sessions.mdx

```diff
@@ -182,139 +224,85 @@ Teleport > Connecting to prod.teleport.example.com over SSH
 
-When joining a session with `tsh join` or `tsh kube join`, a user can specify a
-participant mode with the `--mode <mode>` flag , where the mode is one of `peer`,
-`moderator` or `observer`. By default, the mode is `observer`.
-
-A participant may leave a session with the shortcut `^c` (Control + c) while in observer or
-moderator mode. When in moderator mode, a participant may also forcefully
-terminate the session at any point in time with the shortcut `t`.
-
-### Require policy count
+If you join a session with `tsh join` or `tsh kube join`, you can specify a
+participant mode with the `--mode <mode>` command-line option, where `<mode>` is `peer`,
+`moderator`, or `observer`. The default participant mode is `observer`.
 
-Require policies can have a variable amount of users that need to match the
-filter expression in order to satisfy the policy. The `count` field of a require
-policy is a positive integer value that specifies the minimum amount of users
-this policy requires.
+You can leave a session with the shortcut `^c` (Control + c) while in observer or
+moderator mode. In moderator mode, you can also forcefully terminate the session 
+at any point in time with the shortcut `t`.
 
-### On leave
+### Multifactor authentication
 
-The `on_leave` string option in require policies is used to define what happens when a moderator leaves a session, causing a policy to no longer be satisfied.
+If `per_session_mfa` is set to `true` in role or cluster settings, Teleport requires 
+multifactor authentication checks when starting new sessions. This requirement is
+also enforced for session moderators. Therefore, moderators who want to join a session
+must have configured a Universal 2nd Factor (U2F) or WebAuthn MFA token.
 
-There are two possible actions to take in this scenario:
-- Terminate the session and disconnect all participants, corresponding to the `"terminate"` value.
-- Pause the session and stop any input/output streaming until the policy is satisfied again, corresponding to the `"pause"` value.
+Every 30 seconds, Teleport will prompt users in the terminal session to authenticate in 
+the next 15 seconds. This behavior continues throughout the session to ensure that 
+moderators are always present and watching a given session.
 
-By default, Teleport treats an empty string in this field as the same as `terminate`.
-That is, the session is terminated instantly and all participants are disconnected.
+If no MFA input is received within 60 seconds, the user is disconnected from the
+session, which might cause the session to terminate or pause because a require policy
+is no longer satisfied.
 
-If all require policies attached to the session owner are set to `"pause"`, the session will instead pause
-but the session will remain open. This discards all input from session participants and buffers the most recent output until the session can resume.
+## Session kinds
```

Comment:
Should line 190 link to this section?

Reply:
The session types are included in the table and the section adds minimal additional content, so adding a link might be more distracting to a reader than not.